inference-engine/src/mkldnn_plugin/nodes/proposal_imp.hpp

```diff
@@ -6,6 +6,8 @@
 #include <vector>
 #include <array>
 
+#include "ie_common.h"
+
```

Comment:
If you manage to remove this include it will be perfect.
There is a cpu developer agreement do not use common headers with template(inline) functions inside a cross compiled sources.